packages/flutter_tools/lib/src/flutter_manifest.dart

```diff
@@ -434,6 +450,11 @@ void _validateFlutter(YamlMap yaml, List<String> errors) {
           _validateFonts(kvp.value as YamlList, errors);
         }
         break;
+      case 'licenses':
+        if (kvp.value is! YamlList || kvp.value[0] is! String) {
+          errors.add('Expected "${kvp.key}" to be a list, but got ${kvp.value} (${kvp.value.runtimeType})');
```

Comment:
should this be two different error messages, one if the `kvp.value is! YamlList` and another if `kvp.value[0] is! String`?